demo/webpack.config.js

```diff
@@ -16,7 +34,18 @@ module.exports = {
     path: path.resolve(__dirname, 'dist'),
     filename: '[name].js',
   },
-  plugins: [],
+  plugins: [
+    new HtmlWebpackPlugin({
```

Comment:
images are broken because the plugin adds all entries in the HTML file, meaning the webworker is loaded by the HTML, and then registered by the javascript, what causes the image rendering issue.

Reply:
good point

Reply:
fixed by https://github.com/camptocamp/inkmap/pull/13/commits/c62c223026d959da915f1554f1d79db018ab4bae

now the output files are not appended to the html; this was made to avoid having the worker script loaded in the html

Reply:
See CI PR for quick fix for the demo build (we had to add HTMLPlugin as well for the gh-pages). I think it's acceptable, we'still have some challenges ahead when we will publish the npm package embeding the worker ...

Reply:
it'll be fine :)